src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -452,13 +452,48 @@ public static void updateHearingChannel(SscsCaseData caseData, HearingChannel he
             appeal.setHearingSubtype(hearingSubtype);
         }
 
-        hearingSubtype.setWantsHearingTypeFaceToFace(hearingChannelToYesNoString(FACE_TO_FACE, hearingChannel));
-        hearingSubtype.setWantsHearingTypeTelephone(hearingChannelToYesNoString(TELEPHONE, hearingChannel));
-        hearingSubtype.setWantsHearingTypeVideo(hearingChannelToYesNoString(VIDEO, hearingChannel));
+        if (hearingType == HearingType.PAPER) {
```

Comment:
Do we need to add this if check? `hearingType` is set to PAPER only if the `hearingChannel` is either NOT_ATTENDING or PAPER, and the `wantsHearingTypeFaceToFace`, `wantsHearingTypeTelephone` and `wantsHearingTypeVideo` should be set to `No` anyway in the else clause if the `hearingChannel` is not FACE_TO_FACE, TELEPHONE or VIDEO

Reply:
Not a necessity, but to just make it more robust so when hearing channel is selected to 'Paper', it should set all the hearing subtypes on case details to No, not just change the hearing type. So if a subtype on case details was 'yes' previously, this new hearing channel (paper) should set it to "No" on the case details.